cvat/apps/engine/tests/utils.py

```diff
@@ -189,6 +189,87 @@ def _check_request_status(
         assert request_status == "finished", f"The last request status was {request_status}"
         return response
 
+class ImportApiTestBase(ApiTestBase):
+    def _import(
+        self,
+        user: str,
+        api_path: str,
+        file_content: BytesIO,
+        *,
+        through_field: str,
+        query_params: dict[str, Any] | None = None,
+        expected_4xx_status_code: int | None = None,
+    ):
+        if query_params:
+            assert "?" not in api_path
+            api_path += "?" + urlencode(query_params)
```

Comment:
IIRC, `query_params` can now be passed directly to `_post_request`.

Reply:
Hm, I thought that I updated it. Okay, I'll fix